neqo-transport/src/streams.rs

```diff
@@ -58,6 +111,8 @@ impl Streams {
             local_stream_limits: LocalStreamLimits::new(role),
             send: SendStreams::default(),
             recv: RecvStreams::default(),
+	    ordered_send: BTreeMap::new(),
+	    unordered_send: HashSet::new(),
```

Comment:
I think these two members should be in `SendStreams`.
I prefer putting all send order logic in `SendStream`.